### ISSUE TITLE
Memory usage seems always growing with LRU

### ISSUE THREAD

**Issue by juliusknorr** [nextcloud/whiteboard#313]
At least on prometheus metrics we have a constant growth of memory usage:

﻿﻿![Image](https://github.com/user-attachments/assets/d2c9ec3c-7fde-4648-9558-340e388ed70a)

@hweihwang Any idea, happens on our instance with the latest release

**Comment by hweihwang** [comment]
Thank you! It's highly because of this one https://github.com/nextcloud/whiteboard/pull/259. Let me check in details

**Comment by hweihwang** [comment]
@juliusknorr  Oh this is happen even when https://github.com/nextcloud/whiteboard/pull/259 not released? That might be interesting too

**Comment by juliusknorr** [comment]
Yes, this instance is running the latest release 1.0.4 so #259 is not included
